src/main/java/com/dreamsportslabs/guardian/dao/model/ClientScopeModel.java

```diff
@@ -0,0 +1,18 @@
+package com.dreamsportslabs.guardian.dao.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ClientScopeModel {
```

Comment:
Why do we need so many ways to create this object?
Also its an anti pattern to have setters and Builder pattern both.
Re think the object creation and modification aspect.